### ISSUE TITLE
XML documentation texts should not end with exclamation or question mark

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#462]
Following should trigger a violation:
```c#
/// <summary>
/// Some text!
/// </summary>
```

The reason is that if it is important, it should be placed inside a `<note type="important">` tag, so that the user, when reading the comment (assuming the website), can immediately see that there is something to be aware of.

Following should NOT trigger a violation:
```c#
/// <summary>
/// Some text.
/// </summary>
/// <remarks>
/// <note type="important">
/// Some warning!
/// </note>
/// </remarks>
```

The same is valid for any other tag except of:
- `<note>`
- `<para>` inside a `<note>`
- `<c>`
- `<code>`

**Comment by RalfKoban** [comment]
Beware of links, such as:
    /// https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/loggermessage?view=aspnetcore-6.0.
